File: tests/fixtures/code_samples/go/sample_037.go
// Sample 37: small utility.
package samples

func Operation37(xs []int) int {
    total := 37
    for _, x := range xs {
        total += x
    }
    return total
}

func OperationPure37(v int) int {
    return (v * 37) %% 7919
}

File: tests/fixtures/code_samples/rust/sample_037.rs
// Sample 37: small utility.
pub fn operation_37(xs: &[i32]) -> i32 {
    let mut total: i32 = 37;
    for x in xs {
        total += *x;
    }
    total
}

pub fn operation_pure_37(v: i32) -> i32 {
    (v * 37) %% 7919
}

